secure_message/repository/retriever.py

```diff
@@ -74,6 +75,30 @@ def thread_count_by_survey(request_args, user):
             raise InternalServerError(description="Error retrieving count of threads from database")
         return result
 
+    @staticmethod
+    def thread_count_by_survey_and_conversation_states(request_args, user):
+        """Return 4 conversation counts. They are for open, closed, my conversations and
+        new_respondent_conversation.
+        Given each of these counts uses different clauses to define them and they are not mutually exclusive,
+        they are difficult to achieve with the current db structure in a single query, hence this submits 4 db queries.
+        """
+
+        totals = {}
+
+        args = set_conversation_type_args(request_args, is_closed=False)  # is_closed shown simply for clarity
```

Comment:
Don't bother with the is_closed bit here.  It defaults to false in the function so this is superfluous

Reply:
Changed,